lib/ui/painting/single_frame_codec.h

```diff
@@ -30,7 +31,10 @@ class SingleFrameCodec : public Codec {
   size_t GetAllocationSize() override;
 
  private:
+  enum { STATUS_NEW, STATUS_IN_PROGRESS, STATUS_COMPLETE } status_;
```

Comment:
We follow the enum names as constants pattern as described here https://google.github.io/styleguide/cppguide.html#Enumerator_Names. Please use an enum class instead.

So,

```
enum class Status {
  kNew,
  kInProgress,
  kComplete,
};
```